src/main/java/mate/academy/dto/BookDto.java

```diff
@@ -1,15 +1,13 @@
 package mate.academy.dto;
 
 import java.math.BigDecimal;
-import lombok.Data;
 
-@Data
-public class BookDto {
-    private Long id;
-    private String title;
-    private String author;
-    private String isbn;
-    private BigDecimal price;
-    private String description;
-    private String coverImage;
-}
+public record BookDto(
+        Long id,
+        String title,
+        String author,
+        String isbn,
+        BigDecimal price,
+        String description,
+        String coverImage
+) {}
```

Comment:
don't your Checkstyle fire at it ?)

I thought it is mandatory to have the brackets open...

Reply:
No, everything is ok and all checks have passed